loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py

```diff
@@ -0,0 +1,7 @@
+my_list = [1, 2, 3, 4, 6, 7, 8]
+filtered = []
```

Comment:
We don't use this filtered list. Delete it

Reply:
fixed